backend/authentication/providers/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2017 Sensu Inc.
```

Comment:
Do we need to update the copyright year since we've made changes to this code in 2019?